lib/tasks/invite_code.rake

```diff
@@ -1,16 +1,25 @@
 
 
 desc "Generate a batch of invite codes"
-task :generate_invites, [:n] => :environment do |task, args|
+task :generate_invites, [:n, :u] => :environment do |task, args|
 
-    codes = args.n
-    codes = codes.to_i
+  codes = args.n
+  codes = codes.to_i
 
-    codes.times do |x|
+  if args.u == nil
+    max_uses = 1
+
+  elsif
+    max_uses = args.u
+    max_uses = max_uses.to_i
```

Comment:
The codes.to_i above is needed to the codes.times below works. (times is only defined on integers and not on strings). :thought_balloon: But do we also need it for the max_uses setter? :confused: :grey_question: 
It sure does not do no harm - but maybe we can cut a tiny bit of compexity. :scissors: